web3/auto/infura/loader.py

```diff
@@ -1,24 +1,17 @@
 import logging
 import os
 
-from web3 import (
-    HTTPProvider,
-    Web3,
-)
-
 INFURA_MAINNET_BASE_URL = 'https://mainnet.infura.io'
+INFURA_ROPSTEN_BASE_URL = 'https://ropsten.infura.io'
```

Comment:
Orthogonal but realized these are namespaced with `WEB3_` in front of them which is a convention I think we should stick to.  Otherwise we're likely to have collissions or leakage across library boundaries.

Reply:
I assume you mean for the environment variable instead of these constants, ie~ `INFURA_API_KEY` -> `WEB3_INFURA_API_KEY`

Reply:
Correct.